apps/web/src/components/HeroUpdated/Hero.stories.tsx

```diff
@@ -0,0 +1,98 @@
+import { StoryFn, Meta } from "@storybook/react";
+import UserPlusIcon from "@heroicons/react/24/solid/UserPlusIcon";
+import AcademicCapIcon from "@heroicons/react/24/solid/AcademicCapIcon";
+import ChatBubbleBottomCenterIcon from "@heroicons/react/20/solid/ChatBubbleBottomCenterIcon";
+
+import { allModes } from "@gc-digital-talent/storybook-helpers";
+
+import Hero from "./Hero";
+
+export default {
+  component: Hero,
+  args: {
+    title: "Hero",
+    subtitle: "Subtitle",
+    crumbs: [
+      {
+        label: "Home",
+        url: "#home",
+      },
+      {
+        label: "One",
+        url: "#one",
+      },
+      {
+        label: "Two",
+        url: "#two",
+      },
+    ],
+  },
+  parameters: {
+    chromatic: {
+      modes: {
+        light: allModes.light,
+        "light mobile": allModes["light mobile"],
+        dark: allModes.dark,
+        "light iap": allModes["light iap desktop"],
+        "dark iap": allModes["dark iap desktop"],
+      },
+    },
+  },
+} as Meta<typeof Hero>;
+
+const Template: StoryFn<typeof Hero> = (args) => <Hero {...args} />;
```

Comment:
I suggest adding some sample page content.  Otherwise it's hard to see what's going on with the overlap.

Reply:
Okay, yeah. Alright I think I got something going with 
https://github.com/GCTC-NTGC/gc-digital-talent/pull/11713/commits/f06ee4fcefaeaaab9155b6f8060ff7fcf69cbfb6
https://github.com/GCTC-NTGC/gc-digital-talent/pull/11713/commits/93beebbe29280d3ba28a7d94065bb16a5066b49e

Reply:
It's a good start, but not super realistic.  There's a an overlap story on the old hero.  I've grabbed it and updated it a bit.  Maybe try this?
```tsx
 children: (
    <div
      data-h2-wrapper="base(center, large, x1) p-tablet(center, large, x2)"
      data-h2-background-color="base(foreground)"
      data-h2-radius="base(rounded)"
      data-h2-padding="base(x2, x1)"
      data-h2-shadow="base(s)"
      data-h2-height="base(30vh)"
    >
      <p data-h2-font-size="base(h4)" data-h2-text-align="base(center)">
        Replace Me
      </p>
    </div>
  ),
```